docs/pages/reference/managed-updates-v2.mdx

```diff
@@ -0,0 +1,192 @@
+
+**Example configuration:**
+
+```yaml
+kind: autoupdate_config
+metadata:
+  name: autoupdate-config
+spec:
+  agents:
+    mode: enabled
+    strategy: halt-on-error
+    schedules:
+      regular:
+        - name: dev
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 16
+        - name: stage
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 18
+        - name: prod
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 20
+          wait_hours: 24
+```
+
+In this example, the `dev` group updates first, followed by `stage`, and finally `prod`. The `prod` group will only update if both `dev` and `stage` have updated successfully, and will wait 24 hours after `stage` has completed.
+
+### `time-based` Strategy
+
+The `time-based` strategy is designed for environments where update groups are independent of each other, such as geographical regions or different teams. It allows updates to occur whenever the specified maintenance window is active for a group, regardless of the status of other groups.
+
+**Key characteristics:**
+
+- Groups update independently during their configured maintenance windows
+- No dependency between groups - if prod's window occurs before dev's, prod will update first
+- Maintenance windows are strictly enforced - updates only occur during the specified window duration
+- Does not support the `wait_hours` parameter
+
+**Example configuration:**
+
+```yaml
+kind: autoupdate_config
+metadata:
+  name: autoupdate-config
+spec:
+  agents:
+    mode: enabled
+    strategy: time-based
+    maintenance_window_duration: 2h
+    schedules:
+      regular:
+        - name: us-east
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 4
+        - name: us-west
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 6
+        - name: europe
+          days: ["Mon", "Tue", "Wed", "Thu"]
+          start_hour: 12
+```
+
+In this example, each region will update during its own 2-hour maintenance window,
+regardless of the status of other regions. If a new version becomes available
+when Europe's window is active but before US East's window, Europe will update first.
+
+## Version Behavior During Rollout
+
+Depending on which strategy is selected, the system behaves differently regarding which versions are installed at
+different points in the rollout. Consider a scenario where `start_version` is A and `target_version` is B:
+
+### `halt-on-error` Strategy Version Behavior
+
+| Timing                    | New Agent Installations | Existing Agent Updates |
+|---------------------------|-------------------------|------------------------|
+| Before maintenance window | Install version A       | No updates occur       |
+| During maintenance window | Install version B       | Update to version B    |
+| After maintenance window  | Install version B       | Update to version B    |
```

Comment:
This is the behavior within a single group AFAIU, what about between groups? For example, in your earlier example, it took groups "dev" and "stage" a very long time to upgrade so it's well past "prod"'s start time (e.g. it went into the next day), would prod still upgrade right away or does the timer reset somehow?